boards/arm/nucleo_l432kc/doc/nucleol432kc.rst

```diff
@@ -116,6 +116,10 @@ The Zephyr nucleo_l432kc board configuration supports the following hardware fea
 +-----------+------------+-------------------------------------+
 | PWM       | on-chip    | pwm                                 |
 +-----------+------------+-------------------------------------+
+| CAN       | on-chip    | can                                 |
++-----------+------------+-------------------------------------+
+
+.. note:: CAN feature requires additional CAN transceiver
```

Comment:
The word ``additional`` implies there is a transceiver already there and one more is needed.  Is that what you intended?  If not, then this might need to be written as:
```
.. note:: CAN feature requires a CAN transceiver.
```
and the same change in the other .rst file.

Reply:
ok, done. Txs for the review!